packages/material-ui/src/ButtonGroup/ButtonGroup.test.js

```diff
@@ -0,0 +1,229 @@
+import React from 'react';
+import { assert } from 'chai';
+import {
+  createMount,
+  createShallow,
+  createRender,
+  // describeConformance,
+  getClasses,
+} from '@material-ui/core/test-utils';
+import Button from '../Button/Button';
```

Comment:
```suggestion
import Button from '../Button';
```

which is closer to what the location usercode would import from.